constraintlayout/core/src/main/java/androidx/constraintlayout/core/state/State.java

```diff
@@ -33,7 +33,7 @@
  * Represents a full state of a ConstraintLayout
  */
 public class State {
-
+    public CorePixelDp mDpToPixel;
```

Comment:
Maybe this can be protected?

Reply:
made private 
added separate set and get
made set protected and get package protected